src/components/interfaces/HMI_API.xml

```diff
@@ -1253,20 +1253,20 @@
   </element>
 </enum>
 
-<enum name="SteeringWheelLocation">
```

Comment:
@dev-gh fixed in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215?w=1